src/me/mrCookieSlime/Slimefun/Setup/Messages.java

```diff
@@ -26,8 +26,10 @@ public static void setup() {
 		local.setDefault("messages.usage", "&4Usage: &c%usage%");
 		local.setDefault("messages.not-online", "&4%player% &cis not online!");
 		local.setDefault("messages.not-valid-item", "&4%item% &cis not a valid Item!");
-		local.setDefault("messages.given-item", "&bYou have been given a &7\"%item%\"");
-		local.setDefault("messages.give-item", "&bYou have given %player% a &7\"%item%\"");
+		local.setDefault("messages.not-valid-amount", "&4%amount% &cis not a valid amount!");
+		local.setDefault("messages.too-low-amount", "&cAmount must be higher than 0!");
```

Comment:
I wouldnt make extra messages for "too low", just print out the invalid amount message for everything :p